src/Style/GlobalStyle.css

```diff
@@ -1,49 +1,138 @@
+:root {
+  /* The unicode range for Vietnamese Characters is taken from http://vietunicode.sourceforge.net/charset/ */
+  --vn-unicode-range: U+00C0 - U+00C3, U+00C8 - U+00CA, U+00CC - U+00CD,
+    U+00D2 - U+00D5, U+00D9 - U+00DA, U+00DD, U+00E0 - U+00E3, U+00E8 - U+00EA,
+    U+00EC - U+00ED, U+00F2 - U+00F5, U+00F9 - U+00FA, U+00FD, U+0102 - U+0103,
+    U+0110 - U+0111, U+0128 - U+0129, U+0168 - U+0169, U+01A0 - U+01B0,
+    U+1EA0 - U+1EF9;
+}
+
+/* Default Font */
+
+@font-face {
+  font-family: 'Poppins';
+  src: url('../../assets/fonts/Poppins-Regular.ttf');
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+@font-face {
+  font-family: 'Poppins';
+  src: url('../../assets/fonts/Poppins-Bold.ttf');
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+@font-face {
+  font-family: 'Poppins';
+  src: url('../../assets/fonts/Poppins-Light.ttf');
+  font-weight: 300;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+@font-face {
+  font-family: 'Poppins';
+  src: url('../../assets/fonts/Poppins-Medium.ttf');
+  font-weight: 500;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
 @font-face {
-    font-family: 'Poppins';
-    src: url('../../assets/fonts/Poppins-Regular.ttf');
-    font-weight: normal;
-    font-style: normal;
-    font-stretch: normal;
-    line-height: normal;
-    letter-spacing: normal;
+  font-family: 'Poppins';
+  src: url('../../assets/fonts/Poppins-SemiBold.ttf');
+  font-weight: 600;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
 }
 
 @font-face {
-    font-family: 'Poppins';
-    src: url('../../assets/fonts/Poppins-Bold.ttf');
-    font-weight: bold;
-    font-style: normal;
-    font-stretch: normal;
-    line-height: normal;
-    letter-spacing: normal;
+  font-family: 'Poppins';
+  src: url('../../assets/fonts/Poppins-Black.ttf');
+  font-weight: 900;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
 }
 
+/* Vietnamese Font */
+
 @font-face {
-    font-family: 'Poppins';
-    src: url('../../assets/fonts/Poppins-Light.ttf');
-    font-weight: 300;
-    font-style: normal;
-    font-stretch: normal;
-    line-height: normal;
-    letter-spacing: normal;
+  font-family: 'SVN Poppins';
```

Comment:
Just wondering, does this work if you use the same `font-family` name? If it does, then it might be the case that you don't have to specify multiple font families within `GlintsContainerStyle`.

Reply:
@yjwong Yeap it works! Just had to declare the original poppins font-face after declaring the SVN Poppins font-face so the browser will load the original font by default.

Reply:
Awesome!